rfcs/slider-component.md

```diff
@@ -0,0 +1,121 @@
+# Slider Component
+
+Represents control with sliding axis to choose a single value or range between min and max values.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Slider} from 'baseui/slider';
+
+export default () => <Slider range={[
+  {value: 0, label: '0, min'},
+  {value: 100, label: '100, max'}
+]} />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {Slider} from 'baseui/slider';
+
+export default () => {
+  return <Slider
+    range={[
+      {value: 0, label: '0, min'},
+      {value: 20, label: '20'},
+      {value: 40, label: '40'},
+      {value: 60, label: '60'},
+      {value: 80, label: '80'},
+      {value: 100, label: '100, max'}
+    ]}
+    thumbs={4}
+    initialState={{
+      values: [
+        {value: 20, label: '20'}, {value: 40, label: '40'},
+        {value: 60, label: '60'}, {value: 80, label: '80'},
+      ]
+    }}
+    step={20}
+    onChange={(event, values)=>{console.log(values)}}
+    overrides={{
+      Thumb: props => <div style={{
+        color: 'red',
+        borderRadius: '5px',
+      }}></div>,
+    }}
+    />
+}
+```
+
+## Exports
+
+* `Slider`
+* `StatefulSlider`
+* `StatefulSliderContainer`
+* `StyledRoot`
+* `StyledAxis`
+* `StyledTick`
+* `StyledThumb`
+
+## `Slider` API
+
+* `range: {Array<Point>}` - Required
+  All points present on slider axis. First and last point represent min and max values. Each `Point` can be a value itself (for primitive type) or if `Point` is `Object` it should have `value` and `label`. If more than 2 elements are present in array, they represent all ticks on axis and `step` property is ignored.
+* `thumbs: number` - Optional
+  Amount of thumbs representing how many values can be selected. For even quantity of thumbs selected range is considered in between them. Default is `1`
+* `values: {Array<Point>}` - Optional
```

Comment:
How about `value` (not plural) as a name and `Point | Array<Point>` as a value here?

Reply:
ok